### PR TITLE
Added Image Processing Pipeline sample for GKE

### PR DESCRIPTION
**Pull request by meteatamel**

### CODE REVIEW

File: events/image-processing-pipeline/gke/README.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
+# Image Processing Pipeline - GKE
```
Comment (grayside) [inline events/image-processing-pipeline/gke/README.md:1]: While some brief instructions are okay in GitHub, a full tutorial should be published in the official docs or as a community tutorial.

**Comment by grayside** [comment]
Also, in anticipation of the code, it's not clear to me what about the code would end up being specific to Cloud Run for Anthos. If the code is identical, it would be better to emphasize portability and not segment by platform.

**Comment by meteatamel** [comment]
1. The code is in my Knative repo that I link in the instructions..I didn't want to duplicate it here, as that would be a maintenance nightmare. I didn't want to move here, as it's quite valuable that the sample also runs on open-source Knative. As a middle ground, I decided to keep the code in Knative repo and only provide the necessary config/instructions here specific to Events for Cloud Run GKE.

2. The sample is already under "events" folder, so is it really necessary to have `events/events-image-processing-pipeline`? 

3. If I remove the instructions from `README.md`, I believe it'd be useless. I don't think anyone would be able to take it and figure out what's needed to run it on Events for Cloud Run on GKE. If we turn this into official doc or a community tutorial (not planned yet but might happen), I'll be happy to remove the instructions and simply link to that tutorial but until that happens, I don't see the harm in instructions being in the `README.md`.

**Comment by grayside** [comment]
1. The scope of this repo is maintenance of code samples showcased on cloud.google.com or otherwise used as part of instructions on cloud.google.com, so either used in official docs or a community tutorial. This repo is not a publishing target for tutorial content.

2. (My thought was to remove having a top-level `events/` directory, in-line with the approach being taken in other sample repos.

**Pull request closed without merging**